.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
I love me some hidden files 😵

Reply:
Given that this file isn't used in this repo, I think a better option would be if @jdhoffa adds `.env` to `.git/info/exclude` instead of `.gitignore`

Reply:
I don't understand why that is better?